### ISSUE TITLE
Can you add this to a Github Pages site?

### ISSUE THREAD

**Issue by XxFluffyAsherxX** [ZixiaChen/Eaglercraft-1.12.2#1]
please, it would help a lot!

**Comment by kfcandwatermelon** [comment]
h-hear me out..

**Comment by ZixiaChen** [comment]
Sure I'll do it its going to take some time

**Comment by ZixiaChen** [comment]
I cant seem to do it it just takes me to the readme when I go into the github pages and does not show the index.html

**Comment by XxFluffyAsherxX** [comment]
yeah that's frustration, try putting the readme into a separate branch, I'm not great at coding but it might work, then put the other files in the main branch.

**Comment by XxFluffyAsherxX** [comment]
oops, I meant frustrating

**Comment by ZixiaChen** [comment]
ok

**Comment by ZixiaChen** [comment]
wait how about the license the license then will appear in the GitHub pages

**Comment by XxFluffyAsherxX** [comment]
hmmm try to rename your file to index.html and then code in your assets.epk what the assets are and put that in the same branch.

**Comment by Elite4r** [comment]
If this still isn't fixed, put all files inside the Web folder into the regular github repository. Then it should work.
